src/Og.php

```diff
@@ -117,7 +116,10 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
    *  the OG membership ID and the group ID as the value. If nothing found,
    *  then an empty array.
    */
-  public static function getEntityGroups($entity_type, $entity_id, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
+  public static function getEntityGroups(EntityInterface $entity, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
+    $entity_type = $entity->getEntityTypeId();
```

Comment:
btw, sometimes we call it $entity_type, and sometimes $entity_type_id. Lets go with `$entity_type` and `$bundle`?